### ISSUE TITLE
Importing the date picker locally do not work

### ISSUE THREAD

**Issue by Rolanddoda** [praveenpuglia/vuetify-daterange-picker#38]
I imported these:

```javascirpt
import DateRange from 'vuetify-daterange-picker';
import 'vuetify-daterange-picker/dist/vuetify-daterange-picker.css';
```

And this on components section:

```javascript
export default {
	components: { DateRange.name: DateRange },
```

Obviously this won't work

**Comment by praveenpuglia** [comment]
You need to install it as a plugin. Did you do that? If you do, you won't have to import it in component like so.

https://github.com/praveenpuglia/vuetify-daterange-picker#usage

**Comment by Rolanddoda** [comment]
I followed the instructions to import it only in one component. I don't want to install it globally with Vue.use()

**Comment by praveenpuglia** [comment]
#32 was about similar issue but it's solved now. Can you please post the error you get?

**Comment by markhilton** [comment]
I was able to import component locally with: 
```javascript
export default {
    components: { 'v-daterange': VDateRange },
```
but I am getting errors: Unknown custom element: <v-list>, <v-subheader>, etc.

I have VUE CLI 3 project with TypeScript and Vuetify plugin installed.

**Comment by lbogdan** [comment]
@praveenpuglia Looks like you're only exporting the default (and not the `DateRange` component) from https://github.com/praveenpuglia/vuetify-daterange-picker/blob/master/src/index.js ?

**Comment by praveenpuglia** [comment]
Ah! Thanks a ton @lbogdan. Sorry, I didn't have the time to look into it so far because of busy day job. Would you like to send a PR ? I am more than happy to merge one. :)

**Comment by praveenpuglia** [comment]
Fixed in v2.6.2 - https://github.com/praveenpuglia/vuetify-daterange-picker/releases/tag/v2.6.2
